### PR TITLE
Update on Student hub pricing information, community standard checklist and link to Windows Narrator

### PR DESCRIPTION
**Pull request by juliamuiruri4**
# Description

Update on Student hub pricing information - Microsoft certifications are no longer free for students but are offered at a discounted price. Would be important to set the right expectations for the learner.

Update on community standard checklist - Updates path from Insights > Community to Insights > Community Standards to avoid confusing a new learner. Also included missing items on the checklist

Fixed broken link to Windows Narrator, by correctly appending the tracking code

### CODE REVIEW

**Comment by juliamuiruri4** [comment]
Re-run failed job after merging #1265

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Learn the fundamentals of web development with our 12-week comprehensive course
 
 #### 🧑‍🎓 _Are you a student?_ 
 
-Visit [**Student Hub page**](https://docs.microsoft.com/learn/student-hub/?WT.mc_id=academic-77807-sagibbon) where you will find beginner resources, Student packs and even ways to get a free certificate voucher. This is the page you want to bookmark and check from time to time as we switch out content monthly.
+Visit [**Student Hub page**](https://docs.microsoft.com/learn/student-hub/?WT.mc_id=academic-77807-sagibbon) where you will find beginner resources, Student packs and even ways to get free or discounted academic prices for indistry-recognized certifications. This is the page you want to bookmark and check from time to time as we switch out content monthly.
```
Comment (abrilurena) [inline README.md:19]: What do you think?
```suggestion
Visit [**Student Hub page**](https://docs.microsoft.com/learn/student-hub/?WT.mc_id=academic-77807-sagibbon) where you will find beginner resources, Student packs and various methods to obtain free or discounted academic pricing for industry-recognized certifications. This is the page you want to bookmark and check from time to time as we switch out content monthly.
```

**Review by abrilurena — CHANGES_REQUESTED** [review]
Hi, @juliamuiruri4! just did a suggestion. Please let me know if you have any questions, thanks ✨

**Comment by NileshAttili** [comment]
Sorry but I want to unregister from these emails.

THANK YOU !

**Comment by abrilurena** [comment]
Hi, @NileshAttili! I hope you're doing great. I apologize for any inconvenience. There are a couple of reasons why this might be happening, but it’s most likely because:

- You may be "watching" the repository. In GitHub, you can choose to watch a repository, which means you'll get notifications for all discussions — commits, pull requests, comments, and other updates.

To stop getting these notifications, you can:

- Unwatch the repository: Go to the main page of the repository, click on the 'Watch' button (eye icon) at the top right, and click 'Unwatch'.

Please note: These steps will only stop future notifications. 


Hope this helps, please let me know if you need any help from our end ✨

**Pull request closed without merging**